test/compiler.jl

```diff
@@ -28,6 +32,11 @@ function compiler_testsuite()
         @test !any(stmt->(stmt isa Expr) && stmt.head == :invoke, CI.code)
     end
 
+    let (CI, rt) = @ka_code_typed square(CPU())(zeros(1), zeros(1), ndrange=1)
```

Comment:
I noticed to late that this is just `CPU()` and thus doesn't check the CUDAKernels implementation